test/units/testsuite-29.sh

```diff
@@ -116,6 +118,8 @@ portablectl detach --now --enable --runtime /tmp/minimal_1 minimal-app0
 portablectl list | grep -q -F "No images."
 busctl tree org.freedesktop.portable1 --no-pager | grep -q -F '/org/freedesktop/portable1/image/minimal_5f1' && exit 1
 
+# Need to figure out why this is not working
+# portablectl "${ARGS[@]}" attach --now --runtime --extension /usr/share/app0.raw /etc/app00.raw /usr/share/minimal_0.raw app0
```

Comment:
@bluca do you know why this commented out line would fail? The image is created within test-functions file like for sysext's /usr/share/app0.raw

Reply:
`--extension` is missing